allsky.sh

```diff
@@ -147,15 +145,21 @@ echo "export CAMERA=${CAMERA}" > "${ALLSKY_CONFIG}/autocam.sh"
 # This must be called after CAMERA AUTOSELECT above to refresh the file name.
 source "${ALLSKY_SCRIPTS}/filename.sh"
 
+if [ -d "${ALLSKY_TMP}" ]; then
+	# remove any lingering old image files.
+	rm -f "${ALLSKY_TMP}/${FILENAME}"-202*.${EXTENSION}	# "202" for 2021 and later
```

Comment:
FYI: I went through Y2K with all the problems ;-)

Reply:
Andreas, is there anything wrong with this change?
I also went through Y2K - was working at AT&T, and we did so much testing that there were almost no problems.

Reply:
No, but I always have a stomachache with lines of code with years